shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/controllers/BridgeIngressController.java

```diff
@@ -150,6 +162,18 @@ public DeleteControl cleanup(BridgeIngress bridgeIngress, Context context) {
         return DeleteControl.defaultDelete();
     }
 
+    @Override
+    public Optional<BridgeIngress> updateErrorStatus(BridgeIngress bridgeIngress, RetryInfo retryInfo, RuntimeException e) {
```

Comment:
@manstis For my understanding,  I assume the `RetryInfo` here relates to the configuration of the `KubernetesClient`?

Reply:
Correct. This method is called by k8s should it fail to reconcile a resource within it's "timeout/retry" parameters.